vtex/checkout-ui-tests
# CHK-1061: Re-enable test that failed on weekends

#### What is the purpose of this pull request?

[There are three tests models that used to only fail on the weekends](https://github.com/vtex/checkout-ui-tests/pull/98) (more specifically, during noon). This PR reenables one of them.

There is not really a fix in the code or in the store settings, probably some recent configuration update ended up fixing this 🤷🏾 

#### What problem is this solving?

It's part of getting rid of disabled/broken/outdated tests, similar to https://github.com/vtex/checkout-ui-tests/pull/126

#### How should this be manually tested?

You can take this monitoring execution as proof that it didn't fail during the last weekend noon: [shipping-preview/Delivery_Scheduled Delivery_Pickup - vtexgame1nolean.test.js
| 20/12/2021 14:33:34](https://monitoring.vtex.com/healthcheck/results/detail?id=614df02b-3206-6719-0381-6200bdd4d404&environment=stable&month=2021-12). 

Alternatively, you can search more occurrences of this test on the monitoring or confirm that you can't find this test on #qa-monitoring-stable (which logs all monitoring failures)

#### Observation

The reason why I'm only re-enabling one of the three tests is that I want to make this change gradually. So if the tests fail, they won't trigger the on-call during noon. Hopefully, we can get this done in the next two weeks.
